### PR TITLE
chore: more synth fixes

### PR DESCRIPTION
**Pull request by kolea2**
Fixes `.repo-metadata.json` and regenerate

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Thanks for stopping by to let us know something could be better!
 
 Please run down the following list and make sure you've tried the usual "quick fixes":
 
-  - Search the issues already opened: https://github.com/googleapis/java-bigtable/issues
+  - Search the issues already opened: https://github.com/googleapis/google-cloud-bigtable/issues
```
Comment (igorbernstein2) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think this is wrong

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ If you are still having issues, please include as much information as possible:
    General, Core, and Other are also allowed as types
 2. OS type and version:
 3. Java version:
-4. bigtable version(s):
+4. google-cloud-bigtable version(s):
```
Comment (igorbernstein2) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I don’t think this is right

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (kolea2) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is generated by synthtool based off of .repo-metadata.json. If this is incorrect let's raise an issue at the template level

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (kolea2) [reply .github/ISSUE_TEMPLATE/bug_report.md]: same here

Comment (kolea2) [reply .github/ISSUE_TEMPLATE/bug_report.md]: (fwiw I think this one is ok)

**Comment by kolea2** [comment]
Closing in favor of #37

**Pull request closed without merging**